src/styles/blocks/card.scss

```diff
@@ -0,0 +1,120 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  height: 408px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  transition: $transition-speed;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $main-text-color;
+    margin-top: 40px;
+  }
+
+  &:hover &__title {
+    color: $title-hover-color;
+  }
+
+  &__image {
+    align-self: center;
+
+    width: 160px;
+    height: 134px;
+    margin: 0 auto;
+    display: block;
+  }
+
+  &__code {
+    margin-top: 4px;
+
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    text-align: left;
+    color: $secondary-text-color;
+  }
+
+  &__review {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+
+    margin-top: 16px;
+    width: 166px;
+    height: 16px;
+  }
+
+  &__count-review {
+    display: flex;
+    justify-content: flex-end;
+    align-items: flex-end;
+    width: 53px;
+    height: 16px;
+    color: $main-text-color;
```

Comment:
The variable `$main-text-color` is used here, but it is not defined within this file. Ensure that this variable is defined in a global SCSS file or imported correctly to avoid compilation errors.